.github/workflows/release.yml

```diff
@@ -0,0 +1,136 @@
+name: Release and Propose an Upgrade
+
+env:
+  SUBWASM_VERSION: 0.20.0
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - 'main'
+
+jobs:
+  srtool:
+    if: ${{ github.event.pull_request.merged == true && contains(github.event.pull_request.title, 'release:') }}
+    name: SrTool check on ${{ matrix.chain }} ${{ github.event.inputs.ref }}
+    strategy:
+      fail-fast: false
+      matrix:
+        chain: ["amplitude", "pendulum", "foucoco"]
+        shouldReleaseAmp:
+          - ${{ contains(github.event.pull_request.title, 'amplitude') }}
+        shouldReleasePen:
+          - ${{ contains(github.event.pull_request.title, 'pendulum') }}
+        shouldReleaseFou:
+          - ${{ contains(github.event.pull_request.title, 'foucoco') }}
+        exclude:
+          - shouldReleaseAmp: false
```

Comment:
if the pull request title does not have the chain (foucoco, amplitude, or pendulum) in the title, it will be removed in the list.